src/nft/types/sell/sell.ts

```diff
@@ -59,8 +59,14 @@ export interface Listing {
 
 export interface WalletAsset {
   id?: string
-  image_url: string
-  image_preview_url: string
+  imageUrl: string
+  smallImageUrl: string
+  address: string
+  notForSale: boolean
+  animationUrl: string
+  susFlag: boolean
+  priceInfo: PriceInfo
+  tokenType: TokenType
```

Comment:
Can you move tokenType under `asset_contract`?

Reply:
ah you're right thanks for catching that